app/routes/surveys/utils.js

```diff
@@ -55,6 +58,25 @@ export const DetailNavigation = ({
   </NavigationTab>
 );
 
+export const TokenNavigation = ({
+  title,
+  surveyId,
+  actionGrant
```

Comment:
You can use: `actionGrant = []` to give it a default value. Then you don't have to do `actionGrant && moreCode`